tests/manifolds/test_symmetric_positive_definite.py

```diff
@@ -48,6 +48,11 @@ def test_dist(self):
         # Test triangle inequality
         assert manifold.dist(x, y) <= manifold.dist(x, z) + manifold.dist(z, y)
 
+        # Test exponential metric increasing property
+        # (see equation (6.8) in [Bha2007]).
+        logx, logy = multilogm(x), multilogm(y)
```

Comment:
Admittedly it's a bit tedious, but could you also add these new test cases to the product manifold tests? Also, technically we do not need to use `multilogm` here since `x` and `y` are _not_ from a product manifold.

Reply:
> Admittedly it's a bit tedious, but could you also add these new test cases to the product manifold tests?

Which tests do you want to add in `test_product`? separability, symmetry and triangle inequality?

> Also, technically we do not need to use multilogm here since x and y are not from a product manifold.

Ok, but what can I use instead of `multilogm` ?

Reply:
Yep, I've been thinking that some of the more generic tests we should be able to share between manifolds for sure. That's a bit tricky to achieve with `unittest` though since tests defined in a base class will be run on the base class itself as well for some reason without the `setUp` code of the derived class having run yet. That makes it quite hard to share common tests. One option would be to move our `setUp` code to `__init__`, but that method gets called once per tested method of a class. I'll have to think a little more about our testing architecture at some point. For now I think it's fine to just copy the test cases over to the `TestMultiSymmetricPositiveDefiniteManifold` class.

As for `multilogm`, it's really just a style question I supppose since most `multi*` functions work on regular matrices as well. Personally I would switch to `scipy.linalg.logm` in the non-product manifold case though.